website/docs/docs/build/ref-metrics-in-filters.md

```diff
@@ -0,0 +1,163 @@
+sidebar_label: "Metrics as dimensions with metric filters"
+---
+
+Metrics provide users with valuable insights into their data, such as the number of active users. They also are used to track performance, trend, and help businesses made important decisions. Dimensions, on the other hand, are attributes that help categorize data, such as user type or number of orders placed by a customer.
+
+To make informed business decisions, some metrics need the value of another metric as part of the metric definition.
+
+## Use cases
+Some use cases where using metrics as dimensions might be useful are:
+
+- User segments &mdash; Use the number of orders placed by a user in the last 7 days as a dimension when creating user segments.
+- Churn prediction &mdash; Use the number of support tickets an account submitted in the first 30 days to predict potential churn.
+- Activation tracking &mdash; Define account or user activation based on the specific actions within a set number of days after signing.
+
+## Example
+
+As an example, a Software as a service (SaaS) company that sells a data transformation product wants to count activated account. In this case, the definition of an activated account is an account with more than five data model runs.  
+
+To express this metric in SQL:
+- Write a query to calculate the number of data model runs per account
+- Then count the number of accounts who have more than five data model runs:
+
+```sql
+with data_models_per_user as (
+    select
+        account_id as account,
+        count(model_runs) as data_model_runs
+    from 
+        {{ ref('fct_model_runs') }}
+    group by 
+        account_id
+),
+
+activated_accounts as (
+    select
+        count(distinct account_id) as activated_accounts
+    from 
+        {{ ref('dim_accounts') }}
+    left join 
+        data_models_per_user 
+    on 
+        {{ ref('dim_accounts') }}.account_id = data_models_per_user.account
+    where 
+        data_models_per_user.data_model_runs > 5
+)
+
+select
+    *
+from 
+    activated_accounts
+```
+
+This previous SQL query calculates the number of `activated_accounts` by using the `data_model_runs` metric as a dimension for the user entity. It filters based on the metric value scoped to the account entity. You can express this logic natively in the MetricFlow specification.
+
+### Reference a metric in a filter
+
+You can recreate this pattern in MetricFlow by referencing a metric in the `where` filter for another metric using the `Metric()` object syntax. The function for referencing a metric takes a metric name and exactly one entity:
+
+```yaml
+{{ Metric('metric_name', group_by=['Entity']) }}
+```
+
+Using the same `activated_accounts` metric as an example:
+- Create two semantic models, `model_runs` and `accounts`. 
+- Then create a `measure` and `metric` to count data model runs, and another measure to count users.
+- Specify the foreign entity `account` in the `model_runs` semantic model.
```

Comment:
is the below yaml missing a metric calculating data model runs?

Reply:
It exists! The measure `data_model_runs` has `create_metric: true` so that metric will be created automatically.
Though if we think users aren't aware of that attribute, maybe it would be helpful to add the metric YAML explicitly.

Reply:
oh wow, `create_metric: true` is VERY cool! should we add this in the metrics overview doc? and does it apply to other metrics?